lib/cocoapods-core/specification/dsl.rb

```diff
@@ -1316,6 +1316,28 @@ def subspec(name, &block)
         subspec
       end
 
+      # Represents a test specification for the library. Here you can place all
+      # your unit tests for your podspec along with the test dependencies.
+      #
+      # ---
+      #
+      # @example
+      #
+      #   Pod::Spec.new do |spec|
+      #     spec.name = 'NSAttributedString+CCLFormat'
+      #
+      #     spec.test_spec do |test_spec|
+      #       test_spec.source_files = 'NSAttributedString+CCLFormatTests.m'
+      #       test_spec.dependency 'Expecta'
+      #     end
+      #   end
+      #
+      def test_spec(&block)
```

Comment:
This should maybe be:

```
def test_spec(name = 'Tests', type = :unit, &block)
```
or similar to allow for extension into different test target types in the future.